nifi-nar-bundles/nifi-smb-bundle/nifi-smb-processors/src/main/java/org/apache/nifi/processors/smb/ListSmb.java

```diff
@@ -79,32 +90,31 @@
         "previous node left off without duplicating all of the data.")
 @InputRequirement(Requirement.INPUT_FORBIDDEN)
 @WritesAttributes({
-        @WritesAttribute(attribute = "filename", description = "The name of the file that was read from filesystem."),
-        @WritesAttribute(attribute = "shortname", description = "The short name of the file that was read from filesystem."),
-        @WritesAttribute(attribute = "path", description =
+        @WritesAttribute(attribute = FILENAME, description = "The name of the file that was read from filesystem."),
+        @WritesAttribute(attribute = SHORT_NAME, description = "The short name of the file that was read from filesystem."),
+        @WritesAttribute(attribute = PATH, description =
                 "The path is set to the relative path of the file's directory "
                         + "on filesystem compared to the Share and Input Directory properties and the configured host "
                         + "and port inherited from the configured connection pool controller service. For example, for "
                         + "a given remote location smb://HOSTNAME:PORT/SHARE/DIRECTORY, and a file is being listed from "
                         + "smb://HOSTNAME:PORT/SHARE/DIRECTORY/sub/folder/file then the path attribute will be set to \"sub/folder/file\"."),
```

Comment:
`path` attribute is the same as `absolute.path` but does not contain the filename.
So the documentation is not correct: it is not the "relative path" (compared to `absolute.path`) and no filename should be in the example.